parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -339,6 +347,9 @@ installCNI() {
         os_version="current"
     fi
     os_version="${UBUNTU_RELEASE}"
+    if [[ "${os}" == "${MARINER_OS_NAME}" && "${isKata}" == "true" ]]; then
```

Comment:
should be `"${IS_KATA}"`?

Reply:
ya my bad. If bash script supports checking if a variable is undefined, that will be awesome

Reply:
the `-u` option used with the `set` command will have the script forcibly produce an error when an unbound variable is accessed, though we must not be using that option here (and probably because other things would break otherwise...)